pygmt/datasets/samples.py

```diff
@@ -343,3 +346,18 @@ def load_mars_shape(**kwargs):
         fname, sep="\t", header=None, names=["longitude", "latitude", "radius(m)"]
     )
     return data
+
+
+def _load_earth_relief_holes(**kwargs):
```

Comment:
Pylint reports the following warning. 
```
pygmt/datasets/samples.py:351:0: W0613: Unused argument 'kwargs' (unused-argument)
```
I think we don't need `kwagrs` for new datasets.
```suggestion
def _load_earth_relief_holes():
```

Reply:
I'll check again to confirm after I write the tests, but I think it needs `**kwargs` because of `data = load_func[name](suppress_warning=True)`. It was getting passed a value for `suppress_warning` but wasn't expecting any arguments.

Reply:
I received `TypeError: _load_earth_relief_holes() got an unexpected keyword argument 'suppress_warning'` when I tried running it without `**kwargs`.

Reply:
The `suppress_warning` parameter (or `**kwargs`) was added so that users who call the deprecated, old `load_xxx` functions can silent the deprecation warnings by passing `suppress_warning=True`. Thus, `**kwargs` is required for new private `_load_xxx` functions.

Do you think we should rewrite the code to call the old and new load_xxx functions in different ways:
```
    # old public load functions for backward compatibility
    load_func_old = {
        "bathymetry": load_sample_bathymetry,
        "fractures": load_fractures_compilation,
        "hotspots": load_hotspots,
        "japan_quakes": load_japan_quakes,
        "mars_shape": load_mars_shape,
        "ocean_ridge_points": load_ocean_ridge_points,
        "usgs_quakes": load_usgs_quakes,
    }
    # new private load functions
    load_func = {
        "earth_relief_holes": _load_earth_relief_holes,
    }
    if name in load_func_old:
        data = load_func[name](suppress_warning=True)
    else:
        data = load_func[name]
    return data
```

Reply:
I think that's a good idea. Should that be a separate PR?

Reply:
A separate PR is OK to me.